sql/debit_order.sql

```diff
@@ -10,4 +10,15 @@ CREATE TABLE debit_order (
     debit_order_amount NUMERIC(13, 3) NOT NULL,
     debit_order_created_date CHAR(8) NOT NULL
 );
--- rollback DROP TABLE debit_order
\ No newline at end of file
+-- rollback DROP TABLE debit_order
+
+-- changeset ryanbasiltrickett:add-debit-order-disabled
+ALTER TABLE debit_order
+ADD debit_order_disabled BOOLEAN NOT NULL;
+-- rollback ALTER TABLE debit_order DROP COLUMN debit_order_disabled
+
+-- changeset devwasabi2:alter-debit-order-amount
+ALTER TABLE debit_order
+ALTER COLUMN debit_order_amount TYPE BIGINT USING debit_order_amount::BIGINT,
+ALTER COLUMN debit_order_amount SET NOT NULL;
+-- rollback ALTER TABLE debit_order DROP COLUMN debit_order_amount
```

Comment:
Maybe make it easier and just change the main table and we will just recreate the db from scratch